### ISSUE TITLE
It works well

### ISSUE THREAD

**Issue by jaynetics** [dgollahon/rspectre#81]
Not exactly an issue, but as you asked for feedback in the README ...

I have put three large and old codebases through `rspectre` by now, some of them several times.

That is about 150k lines of specs, and about 2 hours of `rspectre` runtime.

Despite the gnarly state of both the code and the specs, I never encountered any issues with `rspectre` itself - no crashes, no false positives, nor did I manage to find any false negatives that hadn't been caught.

Long story short, it works like a charm 😍 

By the way, it inspired me to write a somewhat similar tool, [`factory_sloth`](https://github.com/jaynetics/factory_sloth), which finds `FactoryBot#create` calls that can be replaced with `#build` or `#build_stubbed`. Maybe it is interesting to you.

Cheers,
J.

**Comment by dgollahon** [comment]
Awesome! 😁 Thank you so much for the kind words and feedback! Really helpful to know it's used/liked.

I don't use `factory_bot` but if I did I would definitely use that!

Thanks for your work on `regexp_parser` too, btw. Really cool that it powers regex mutations in [`mutant`](https://github.com/mbj/mutant). I also just saw `repper` which I'm now curious to play with.
